community-membership.md

```diff
@@ -1,77 +1,199 @@
-- **Project Approvers**: approver in [top-level OWNERS file in kubernetes repo](https://github.com/kubernetes/kubernetes/blob/master/OWNERS);
-  de-facto project decision makers; technically can 
-  approve virtually any PRs; can sponsor incubator repos; can sponsor maintainers;
-  maintainer in good standing for at least 1 year; strong technical vision;
-  committed to project's mission and culture; nomination/application process TBD
-- [**API Approver**](https://github.com/orgs/kubernetes/teams/api-approvers):
-  lead designers of the project, who are familiar with the 
-  design, requirements, mechanics, conventions, style, scope, gotchas, etc. 
-  of the API; most beta/GA API changes are vetted by the API approvers
-- [**API Reviewer**](https://github.com/orgs/kubernetes/teams/api-reviewers):
-  contributors familiar with design, requirements, mechanics, conventions, style,
-  scope, gotchas, etc. of the API; have written and/or reviewed Kubernetes APIs
\ No newline at end of file
+## Org member
+
+Org members have issues and PRs assigned to them and participate in SIGs.
+Members are expected to remain active contributors to the community.
+
+### Membership requirements
+
+- Enabled [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication/) on their GitHub account
+- Have contributed to the project or community.  Contribution may include, but is not limited to:
+  - Authoring or reviewing PRs on GitHub
+  - Filing or commenting on issues on GitHub
+  - Contributing to SIG or community discussions (e.g. meetings, slack, email discussion forums, stackoverflow, meetups)
+- Have 2 existing members who will sponsor
+- Have read the [developer guide](contributors/devel/README.md)
+- Subscribed to kubernetes-dev@googlegroups.com
+- Commit to working on a specific area or project
+
+**Note:** New membership is provisional for 3 months, contingent on continued involvement with the project and community
+
+### Responsibilities and privileges
+
+- Tests are automatically run for PullRequests from org members
+- Responsive to issues and PRs assigned to them
+- Responsive to mentions of SIG teams they are membrs of
+- Active owner of code they have contributed (until ownership is transferred)
+  - Code is well tested
+  - Tests consistently pass
+  - Addresses bugs or issues discovered after code is accepted
+  
+**Note:** Org members who frequently contribute code are expected to proactively perform code reviews and work towards
+becoming a primary *code reviewer* for the area that they are active in.
+  
+## Code reviewer
+
+Code reviewers are able to review code for quality and correctness on some part of the repo.
+They are both knowledgeable about the codebase and software engineering principles.
+
+Reviewer status is scoped to a part of the codebase.
+
+**Note:** Acceptance of code contributions requires both a reviewer `lgtm` and approver `approved` on the PR.
```

Comment:
I'm not sure this is needed in this particular doc, as it's more about the PR flow. Reviewers could also be nominated for not k/k, and may have slightly different processes.